### PR TITLE
Networkevents feature is DevPreview

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: docs/FlowCollector.md [hunk 1]
```diff
@@ -5958,8 +5958,8 @@ the kernel debug filesystem, so the eBPF pod has to run as privileged.
 If the `spec.agent.ebpf.privileged` parameter is not set, an error is reported.<br>
 - `DNSTracking`: enable the DNS tracking feature.<br>
 - `FlowRTT`: enable flow latency (sRTT) extraction in the eBPF agent from TCP traffic.<br>
-- `NetworkEvents`: enable the Network events monitoring feature.  This feature requires mounting
-the kernel debug filesystem, so the eBPF pod has to run as privileged.<br/>
+- `NetworkEvents`: enable the Network events monitoring feature. This feature requires mounting
+the kernel debug filesystem, so the eBPF pod has to run as privileged. It is currently a Developer Preview.<br><br/>
```
Comment (skrthomas) [inline docs/FlowCollector.md:5962]:
```suggestion
the kernel debug filesystem, so the eBPF pod has to run as privileged. [IMPORTANT]
====
`NetworkEvents` is available as a Developer Preview.
 ====
 <br><br/>
```

Comment (skrthomas) [reply docs/FlowCollector.md:5962]: Or, minimally, if the admonition doesn't translate between the source and asciidoc, just using all caps IMPORTANT:

**Review by stleerh — COMMENTED** [review]
It should mention that it requires OVN support and to check that your version supports it.

**Comment by jotak** [comment]
@skrthomas @stleerh I changed the text to:

> It requires using the OVN-Kubernetes network plugin with the Observability feature.
IMPORTANT: this feature is available as a Developer Preview.

**Review by stleerh — APPROVED** [review]
/lgtm

**Pull request merged**